### ISSUE TITLE
Could not validate soure - invalid argument

### ISSUE THREAD

**Issue by wilber82** [img2tab/macos-guest-virtualbox#45]
Same issue described in #11 . The bug was closed but I still running into same issue.
The different is script downloaded dmg file successfully.

**Comment by wilber82** [comment]
![2](https://user-images.githubusercontent.com/14928046/55100856-d1e67480-50fd-11e9-92f7-f305e6b84750.PNG)

**Comment by wilber82** [comment]
![1](https://user-images.githubusercontent.com/14928046/55100882-dc087300-50fd-11e9-8243-a826b937cbb2.PNG)

**Comment by wilber82** [comment]
I did image scan. Looks like the image on Apple website has problem? I have tried lot of times.

![3](https://user-images.githubusercontent.com/14928046/55101165-749ef300-50fe-11e9-8c0d-2a641e427b1c.PNG)

**Comment by dshynkev** [comment]
You will probably find that this is the case:
```sh
[-base-3.2# cat /Volumes/Mojave/BaseSystem.dmg
Not Found
[-base-3.2#
```
So it definitely is a problem with the website's response.

In my case, retrying the curl manually a few times eventually fixed it (?!)

**Comment by img2tab** [comment]
Apple released macOS 10.14.4, I need to update the download URL.
Actually this gives me an excuse to implement #33 so I'll get to it.
